tool/ide/RustManifestSyncer.kt

```diff
@@ -141,22 +169,33 @@ class RustManifestSyncer : Callable<Unit> {
 
         private fun attachTestAndBuildProperties(properties: Collection<TargetProperties>) {
             val TESTS_DIR = "tests"
+            val BENCHES_DIR = "benches" // we translate Bazel Tests in 'benches' into Cargo benchmarks
```

Comment:
Bazel `rust_test` targets in the 'benches' directory are added into the parent Cargo.toml so they can be executed by cargo directly.